script.js

```diff
@@ -0,0 +1,53 @@
+let Books = [];
+
+const addBook = () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const book = { id: new Date().getTime(), title, author };
+  if (localStorage.getItem('booksData') !== null) {
+    Books = JSON.parse(localStorage.getItem('booksData'));
+    Books.push(book);
+    const convertedBooks = JSON.stringify(Books);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  } else {
+    Books.push(book);
+    const convertedBooks = JSON.stringify(Books);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  }
+};
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
+  const booksData = localStorage.getItem('booksData');
+  const convertedBooks = JSON.parse(booksData);
+  const remainingBooks = convertedBooks.filter((book) => book.id !== id);
+  const removedBooks = JSON.stringify(remainingBooks);
+  localStorage.setItem('booksData', removedBooks);
+  window.location.reload();
+}
+
+const displayBooks = () => {
+  const booksData = localStorage.getItem('booksData');
+  const convertedBooks = JSON.parse(booksData);
+  document.getElementById('bookstatus').innerHTML = 'No books added';
+  if (convertedBooks && convertedBooks.length === 0) {
+    document.getElementById('bookstatus').innerHTML = 'No books added';
+  } else {
+    document.getElementById('bookstatus').innerHTML = '';
+    let html = '';
+    convertedBooks.forEach((book) => {
```

Comment:
- [ ] During the initial load of the page, the `convertedBooks` data does not exist in the local storage. As a result the `forEach` method would throw an error as `convertedBooks` is evaluated as `null`. Kindly fix this issue. :pray: 

![Screenshot from 2021-11-30 13-45-37](https://user-images.githubusercontent.com/64894927/144033561-3c3bbfc1-1781-4303-a913-31cb5c11cd50.png)